allauth/socialaccount/providers/oauth2/urls.py

```diff
@@ -1,11 +1,20 @@
 from django.conf.urls import patterns, url, include
 
+from allauth.account import app_settings
+from allauth.socialaccount.providers.oauth2.views import proxy_login_callback
 
 def default_urlpatterns(provider):
     urlpatterns = patterns(provider.package + '.views',
                            url('^login/$', 'oauth2_login',
                                name=provider.id + "_login"),
                            url('^login/callback/$', 'oauth2_callback',
-                               name=provider.id + "_callback"))
+                               name=provider.id + '_callback'))
 
+    if app_settings.LOGIN_PROXY_REDIRECT_WHITELIST:
+        urlpatterns += patterns('',
```

Comment:
this doesn't rebase well with the new `django-allauth` version which has removed the `patterns` pattern.  We'll have to find another way in order to support Django 1.9